conference/react-js/src/Routers.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import {
+  BrowserRouter as Router,
+  Route,
+  Link,
+} from 'react-router-dom';
+import Upcomming from './components/Upcomming';
+import Past from './components/Past';
+
+const Routers = () => (
+  <Router>
+    <div className="Container home">
```

Comment:
```Container home``` looks confusing. Why one of those classes starts with a capital letter?

Reply:
these classes was copied from old version application
I will check if need these classes for styles or that support classes from angular